starknet_py/net/models/transaction.py

```diff
@@ -241,7 +241,7 @@ def compute_invoke_hash(
 
     return compute_transaction_hash(
         tx_hash_prefix=TransactionHashPrefix.INVOKE,
-        contract_address=contract_address,
+        contract_address=sender_address,
```

Comment:
I don't know, it hasn't been updated in cairo-lang :/

Reply:
They will probably change it one day. I think `sender_address` is better

Reply:
Putting
```
sender_address=contract_address
```
when calculating the hash for DEPLOY_ACCOUNT transactions seems weird to me.

Reply:
Let's leave it as `contract_address` for now - one less breaking change. We can change it once all transactions are using `sender_address` instead of `contract_address`.